src/components/App.js

```diff
@@ -3,11 +3,13 @@ import * as Babel from "@babel/standalone";
 
 import { CustomPlugin } from "./CustomPlugin";
 import { MainMenu } from "./MainMenu";
+import { Forks } from "./Forks";
 import { Input } from "./Input";
 import { Output } from "./Output";
 import { gzipSize } from "../gzip";
 import { Root } from "./styles";
 import { useDebounce } from "../utils/useDebounce";
+import REPLState from "../state/REPLState";
```

Comment:
Can we remove the imports that aren't being used anymore (along with the comments on line 95-98)?